<!-- Copyright (c) 2017, United States Government, as represented by the     -->
<!-- Administrator of the National Aeronautics and Space Administration.     -->
<!--                                                                         -->
<!-- All rights reserved.                                                    -->
<!--                                                                         -->
<!-- The Astrobee platform is licensed under the Apache License, Version 2.0 -->
<!-- (the "License"); you may not use this file except in compliance with    -->
<!-- the License. You may obtain a copy of the License at                    -->
<!--                                                                         -->
<!--     http://www.apache.org/licenses/LICENSE-2.0                          -->
<!--                                                                         -->
<!-- Unless required by applicable law or agreed to in writing, software     -->
<!-- distributed under the License is distributed on an "AS IS" BASIS,       -->
<!-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         -->
<!-- implied. See the License for the specific language governing            -->
<!-- permissions and limitations under the License.                          -->
<launch>
  <arg name="topic" default="/hw/depth_haz/extended"/>
  <arg name="topic_type" default="extended"/>
  <arg name="amplitude_factor" default="100.0"/>
  <node name="pico_proxy" pkg="pico_driver" type="pico_proxy" output="screen">
    <param name="topic" type="string" value="$(arg topic)"/>
    <param name="topic_type" type="string" value="$(arg topic_type)"/>
    <param name="amplitude_factor" type="double" value="$(arg amplitude_factor)"/>
  </node>
</launch>
